proposals/secure-storage.md

```diff
@@ -0,0 +1,158 @@
+# browser.secureStorage
+
+WebExtension proposal to allow the secure storage of data in platform specific locations.
+
+## Champions
+
+- [@oliverdunk](https://github.com/oliverdunk) (1Password) - oliver@1password.com
```

Comment:
```suggestion
- Oliver Dunk (1Password) - [@oliverdunk](https://github.com/oliverdunk), [email](oliver@1password.com)
```

The downside of the change made in response to @Rob--W's [comment](https://github.com/w3c/webextensions/pull/167#discussion_r812960247) is that we lost your name. Perhaps we could tweak the formatting a bit to keep your name and include your GitHub username. WDYT?

Reply:
I'm happy with that! I accepted your suggestion and tweaked the email, let me know what you think. I can't merge this but I think it's good to go, unless you want anyone else to review.